docs/specifications/tests/DNSSEC-TP/dnssec15.md

```diff
@@ -0,0 +1,176 @@
+# DNSSEC15: Existence of CDS and CDNSKEY
+
+## Test case identifier
+**DNSSEC15**
+
+## Objective
+
+CDS and CDNSKEY record types are defined in [RFC 7344] and [RFC 8878].
+Both record types are optional in a zone. The objective of this test
+case is to verify that they are correctly set-up, if included in the
+zone.
+
+If an CDS record is included in the zone, the corresponding CDNSKEY
+record should also be included ([RFC 7344][RFC 7344, section 4],
+section 4).
+
+The CNS and CDNSKEY RRsets should be consistent between all name
+servers for the zone in question.
+
+If there are both CDS RRs and CDNSKEY RRs in the zone they must match in 
+content ([RFC 7344][RFC 7344, section 4], section 4). It means that both
+must be derived from the same DNSKEY or both being "delete" CDS and
+CDNSKEY.
```

Comment:
Does the RFC really say that?
Section 4.1:
Signer: MUST be signed with a key that is represented in both the
      current DNSKEY and DS RRsets, unless the Parent uses the CDS or
      CDNSKEY RRset for initial enrollment; in that case, the Parent
      validates the CDS/CDNSKEY through some other means
```
Does this mean that there could be 2 KSK/2 ZKS and differnt CDS/CDNSKEY on different authoritatives but still be valid?

Reply:
I am not sure what you mean. From the RFC:

> If the Child publishes both, the two RRsets MUST match in content.